python/GangaDirac/Dockerfile

```diff
@@ -1,34 +1,28 @@
-FROM alexanderrichards/dirac_ui:latest
+# FROM taking build arg requires at least
+#   Docker version 17.05.0-ce, build 89658be
+ARG ganga_version=latest
+FROM ganga-core:$ganga_version
```

Comment:
It should be possible to build the images as part of the release process. If we do it in the first step of the release process, then we can actually run the tests against these images.